requirements.txt

```diff
@@ -0,0 +1,49 @@
+# This file may be used to create an environment using:
+# $ conda create --name <env> --file <this file>
+# platform: osx-64
+brotlipy=0.7.0=py39hcbf5805_1001
```

Comment:
How was this file created? I've not seen versioning with the py39... 

Also, please use less strict versioning. Typically we just version on major releases unless we know that we need a minor version for some reason. See https://www.python.org/dev/peps/pep-0440/#version-specifiers